Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -398,6 +600,231 @@ def object_provision_command(client: PolicyComputeEngine, args: Dict[str, Any])
+) -> CommandResults:
+    """Create an enforcement boundary.
+
+    Args:
+        client: PolicyComputeEngine to use.
+        args: Arguments passed with the command.
+
+    Returns:
+        CommandResult object
+    """
+    name = args.get("name")
+    port = args.get("port")
+    protocol = args.get("protocol", "tcp").lower()
+    providers = args.get("providers")
+    consumers = args.get("consumers")
+
+    validate_required_parameters(
+        name=name, port=port, providers=providers, consumers=consumers
+    )
+    providers = argToList(providers)
+    consumers = argToList(consumers)
+    port = arg_to_number(port, arg_name="port")  # type: ignore
+    validate_enforcement_boundary_create_arguments(port, protocol)  # type: ignore
+    proto = convert_protocol(protocol)
+
+    enforcement_boundary_rule = EnforcementBoundary.build(
+        name=name,
+        consumers=consumers,  # type: ignore
+        providers=providers,  # type: ignore
+        ingress_services=[{"port": port, "proto": proto}],
+    )
+    enforcement_boundary = client.enforcement_boundaries.create(enforcement_boundary_rule)  # type: ignore
+
+    enforcement_boundary_dict = enforcement_boundary.__dict__
+    enforcement_boundary_json = enforcement_boundary.to_json()
+    enforcement_boundary_json["href"] = enforcement_boundary_dict.get("href")
+    enforcement_boundary_json["created_at"] = enforcement_boundary_dict.get(
+        "created_at"
+    )
+    enforcement_boundary_json["updated_at"] = enforcement_boundary_dict.get(
+        "updated_at"
+    )
+    enforcement_boundary_json["deleted_at"] = enforcement_boundary_dict.get(
+        "deleted_at"
+    )
+    enforcement_boundary_json["created_by"] = enforcement_boundary_dict.get(
+        "created_by"
+    )
+    enforcement_boundary_json["updated_by"] = enforcement_boundary_dict.get(
+        "updated_by"
+    )
+    enforcement_boundary_json["deleted_by"] = enforcement_boundary_dict.get(
+        "deleted_by"
+    )
+    enforcement_boundary_json["update_type"] = enforcement_boundary_dict.get(
+        "update_type"
+    )
+    enforcement_boundary_json["caps"] = enforcement_boundary_dict.get("caps")
+
+    readable_output = prepare_enforcement_boundary_create_output(
+        enforcement_boundary_json
+    )
+
+    return CommandResults(
+        outputs_prefix="Illumio.EnforcementBoundary",
+        readable_output=readable_output,
+        outputs_key_field="href",
+        raw_response=enforcement_boundary_json,
+        outputs=remove_empty_elements(enforcement_boundary_json),
+    )
+
+
+def update_enforcement_mode_command(
+    client: PolicyComputeEngine, args: Dict[str, Any]
+) -> CommandResults:
+    """Update enforcement mode for one or more workloads.
+
+    Args:
+        client: PolicyComputeEngine to use.
+        args: arguments obtained from demisto.args()
+
+    Returns:
+        CommandResult object
+    """
+    enforcement_mode: str | None = args.get("enforcement_mode")
```

Comment:
As above, please remove this type union syntax altogether as it requires python 3.10+

Reply:
Removed